ReactWindows/ReactNative.Shared/Modules/AppState/AppStateModule.cs

```diff
@@ -1,18 +1,26 @@
-﻿using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Linq;
 using ReactNative.Bridge;
 using ReactNative.Modules.Core;
+using System.Collections.Generic;
 
 namespace ReactNative.Modules.AppState
 {
     /// <summary>
     /// Native module for monitoring the state of the application.
     /// </summary>
-    public class AppStateModule : ReactContextNativeModuleBase, ILifecycleEventListener
+    public class AppStateModule : ReactContextNativeModuleBase, IBackgroundEventListener
     {
```

Comment:
Should we split this implementation so WPF has it's own AppStateModule?

Reply:
Are you suggesting copying the file as it was to WPF and only applying these changes to the UWP implementation? Because wpf doesn't have the leaving/entered background events?

Reply:
That seems to be a reasonable solution for now. I suspect that we'll need a different implementation for this for WPF. If you think there's a better way, by all means...

Reply:
The state machine has code like this:
```
#if WINDOWS_UWP
        private static bool s_isBackroundModeSupported = ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 3);
#else
        private static bool s_isBackroundModeSupported = false;
#endif
```

So WPF works correctly, if that means supporting just "active" state because "background" doesn't exist.